### ISSUE TITLE
No notification on backup failure

### ISSUE THREAD

**Issue by Standarduser** [simatec/ioBroker.backitup#770]
Due to a misconfiguration of my Dropbox account BackItUp was not able to upload backup files.
I got following error messages in log:

```
2022-10-19 01:30:21.734 - error: backitup.0 (463496) [ccu/dropbox] upload Dropbox: {"error_summary":"path/conflict/file/.","error":{".tag":"path","reason":{".tag":"conflict","conflict":{".tag":"file"}},"upload_session_id":"pid_upload_session:xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"},"code":409}
--
2022-10-19 02:00:30.150 - error: backitup.0 (463496) [iobroker/dropbox] upload Dropbox: {"error_summary":"path/conflict/file/...","error":{".tag":"path","reason":{".tag":"conflict","conflict":{".tag":"file"}},"upload_session_id":"pid_upload_session: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"},"code":409}
```

Although telegram notification (only errors) is configured I got no notification.

**Comment by michipi85** [comment]
i have a similar problem. A backup was not completed because it could not connect to my NAS (wrong settings) so no notification was triggered.

**Comment by simatec** [comment]
Fix in v2.5.4
